robocorp-code/src/robocorp_code/inspector/inspector__main__.py

```diff
@@ -12,7 +12,7 @@
 
 
 def _stderr_reader(stream):
-    from robocorp_ls_core.robotframework_log import get_logger
+    from robocorp_ls_core.robotframework_log import get_logger  # type: ignore
```

Comment:
This shouldn't be needed. You have to configure your IDE to pick up things properly or run `python -m dev vendor_robocorp_ls_core`.

Reply:
That was added because of some lints failing. Maybe they were just indicated on my end.

Reply:
Yes, that should definitely work as the types are there in these cases (if it's not working in your end it's some misconfiguration, if needed we can do a zoom to diagnose).

Reply:
OK. Removed